project_double_alias/views/project_project_view.xml

```diff
@@ -0,0 +1,54 @@
+<odoo>
+
+    <record id="edit_project" model="ir.ui.view">
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project"/>
+        <field name="arch" type="xml">
+            <label for="alias_name" position="attributes">
+                <attribute name="string">Email Alias (Tasks)</attribute>
+                <attribute name="attrs">{'invisible': [('use_tasks', '=', False)]}</attribute>
+            </label>
+            <label for="alias_model" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </label>
+            <field name="alias_model" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <div name="alias_def" position="attributes">
+                <attribute name="attrs">{'invisible': [('use_tasks', '=', False)]}</attribute>
+            </div>
+            <div name="alias_def" position="after">
+                <label for="second_alias_name"
+                       string="Email Alias (Issues)"
+                       attrs="{'invisible': [('use_issues', '=', False)]}"
+                />
+                <div name="second_alias_def" attrs="{'invisible': [('use_issues', '=', False)]}">
+                    <field name="second_alias_id" class="oe_read_only oe_inline" />
+                    <div class="oe_edit_only oe_inline" name="edit_second_alias" style="display: inline;" >
+                        <field name="second_alias_name" class="oe_inline"/>@<field name="alias_domain2" class="oe_inline"/>
+                    </div>
+                </div>
+            </div>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_project_kanban">
```

Comment:
It doesn't matter. We can change it only for readability. In fact, I copy/paste the Odoo view, but you know that they don't really follow conventions at all.